### ISSUE TITLE
Query need to return null values without casting

### ISSUE THREAD

**Issue by mmafic** [HadesArchitect/GrafanaCassandraDatasource#161]
It looks like currently all values are being cast to float64, is it possible to return values from a query without casting? With all null values being coalesced to 0 it's impossible to get a minimum value in the dataset, which breaks some use cases.

**Comment by HadesArchitect** [comment]
@unflag This one was fixed with your #172?

**Comment by unflag** [comment]
No, nil values are not supported yet. I'll look into this, perhaps it may fit well into the current changes.

**Comment by unflag** [comment]
There is a limitation on a `gocql` level - `MapScan` method doesn't support nil values, unfortunately.
https://github.com/gocql/gocql/issues/1699
There is a possible workaround mentioned in that issue - I'll try to figure it out.

**Comment by HadesArchitect** [comment]
@unflag was this one closed by 3.0.0?

**Comment by unflag** [comment]
No, it does not, unfortunately.
